js/modules/k6/http/request.go

```diff
@@ -437,21 +437,25 @@ func (c *Client) prepareBatchObject(requests map[string]interface{}) (
 
 // Batch makes multiple simultaneous HTTP requests. The provideds reqsV should be an array of request
 // objects. Batch returns an array of responses and/or error
-func (c *Client) Batch(reqsV goja.Value) (interface{}, error) {
+func (c *Client) Batch(reqsValues ...goja.Value) (interface{}, error) {
```

Comment:
```suggestion
func (c *Client) Batch(reqsV ...goja.Value) (interface{}, error) {
```
Removing the line after we can restore the name so we don't need to update the comment.

Reply:
Thanks a lot for your valuable feedback. Implemented the change.